### PR TITLE
Submitting oo final

### PR DESCRIPTION
**Pull request by cbrown285**

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
@cbrown285 it appears that your code might be in zip file?

can you please commit individual files so we can review the code?

**Comment by cbrown285** [comment]
Hi Natasha,

I apologize. I hadn't realize I uploaded the zip file instead of the python file. I just uploaded the code a moment ago

Best,

Chris

**Comment by natasha-aleksandrova** [comment]
much better thank you!

File: students/cbrown/Lesson 9/mailroom_oo_final.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+class Donor_Collection():
+
+    donor_dict = {"Bill Gates": [539000, 235642],
+          "Jeff Bezos": [108356, 204295, 897345],
+          "Satya Nadella": [236000, 305352],
+          "Mark Zuckerberg": [153956.35],
+          "Mark Cuban":[459035, 369.50, 570.89]}
+
+    def __init__(self, donor=None):
+        self.donor = donor
```
Comment (natasha-aleksandrova) [inline students/cbrown/Lesson 9/mailroom_oo_final.py:13]: the relationship here is incorrect. DonorCollection should reference a collection of donor objects.

self.donor represents a single Donor object which is not a valid relationship.

File: students/cbrown/Lesson 9/mailroom_oo_final.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
+import sys
+
+
+class Donor_Collection():
+
+    donor_dict = {"Bill Gates": [539000, 235642],
+          "Jeff Bezos": [108356, 204295, 897345],
+          "Satya Nadella": [236000, 305352],
+          "Mark Zuckerberg": [153956.35],
+          "Mark Cuban":[459035, 369.50, 570.89]}
```
Comment (natasha-aleksandrova) [inline students/cbrown/Lesson 9/mailroom_oo_final.py:10]: donor_dict should not be a class attribute but instead should be an instance attribute (defined in `__init__`)

File: students/cbrown/Lesson 9/mailroom_oo_final.py [hunk 3]
```diff
@@ -0,0 +1,224 @@
+import sys
+
+
+class Donor_Collection():
```
Comment (natasha-aleksandrova) [inline students/cbrown/Lesson 9/mailroom_oo_final.py:4]: class names should have no underscores -> `DonorCollection`

**Review by natasha-aleksandrova — COMMENTED** [review]
see comments please.

**Pull request merged**